buildenv/jenkins/JenkinsfileBase

```diff
@@ -811,13 +811,17 @@ def run_parallel_tests() {
 					def jobInvocation = cjob.value.getRawBuild()
 					def buildId = jobInvocation.getNumber()
 					def name = cjob.value.getProjectName()
+					def result = cjob.value.getCurrentResult()
 					try {
+						echo "${name} #${buildId} completed with status ${result}"
 						copyArtifacts (projectName: "${name}", selector: specific("${buildId}"), filter: "**/TestTargetResult.tap", target:"${name}/${buildId}")
+						step([$class: "TapPublisher", testResults: "${name}/${buildId}/**/TestTargetResult.tap", outputTapToConsole: false, failIfNoResults: false])
 					} catch (Exception e) {
 						echo "Cannot copy TestTargetResult.tap from ${name} with buildid ${buildId} . Skipping copyArtifacts..."
+						echo "set build status to ${result}"
+						currentBuild.result = result
```

Comment:
If we have multiple exception, this will reset the result multiple times. Please move this outside of the loop.